crates/iota-transaction-builder/src/lib.rs

```diff
@@ -466,6 +466,39 @@ impl TransactionBuilder {
         Ok(())
     }
 
+    /// Add a single move call to the provided
+    /// [`ProgrammableTransactionBuilder`].
+    /// Accepting PtbInput so one can also provide results from previous move
+    /// calls.
```

Comment:
Nit: To comply with the [summary-sentence guidelines](https://github.com/rust-lang/rfcs/blob/master/text/1574-more-api-documentation-conventions.md#summary-sentence)

```suggestion
    /// Adds a single move call to the provided
    /// [`ProgrammableTransactionBuilder`].
    ///
    /// Accepting [`PtbInput`] so one can also provide results from previous move
    /// calls.
```